README.md

```diff
@@ -167,6 +167,36 @@ For detailed info about the logic and usage patterns of Example42 modules read R
           my_class => 'apache::example42',
         }
 
+## USAGE - Hiera Support
+* Manage apache configuration using Hiera
+
+```yaml
+apache_template: 'modules/apache/apache2.conf.erb'
```

Comment:
I'd suggest to use normal datababindings naming scheme instead of the params_lookup one (they both work with nextgen mods):
apache::template ... instead of apache_template